api_app/views.py

```diff
@@ -117,6 +104,77 @@ def _analysis_request(
     )
 
 
+def _multi_observable_analysis_request(
+    request,
+    serializer_class: ObservableAnalysisSerializer,
+):
+    """
+    Prepare and send multiple observables for analysis
+    """
+    logger.info(
+        f"_analysis_request {serializer_class} received request from {request.user}."
+        f"Data:{dict(request.data)}."
+    )
+
+    # serialize request data and validate
+    serializer = serializer_class(data=request.data, many=True)
+    serializer.is_valid(raise_exception=True)
+
+    serialized_data = serializer.validated_data
+    print(serialized_data)
+    runtime_configurations = [
+        data.pop("runtime_configuration", {}) for data in serialized_data
+    ]
+
+    # save the arrived data plus new params into a new job object
+    jobs = serializer.save(
+        user=request.user,
+    )
+
+    logger.info(f"New Jobs added to queue <- {repr(jobs)}.")
+
+    # Check if task is test or not
+    if not settings.STAGE_CI:
+        # fire celery task
+        for index, job in enumerate(jobs):
+            runtime_configuration = runtime_configurations[index]
+            celery_app.send_task(
+                "start_analyzers",
+                kwargs=dict(
+                    job_id=job.pk,
+                    analyzers_to_execute=job.analyzers_to_execute,
+                    runtime_configuration=runtime_configuration,
+                ),
+            )
+
+    ser = AnalysisResponseSerializer(
+        data=[
+            {
+                "status": "accepted",
+                "job_id": job.pk,
+                "warnings": serialized_data[index]["warnings"],
```

Comment:
ok I got you inserted warning correctly here, I was thinking about the `_analysis_request` view

Reply:
:)